app/posts/personalised-prevention-platform/2025/04/2025-04-21-alpha.md

```diff
@@ -0,0 +1,253 @@
+---
+title: "Onboarding users"
+description: "Exploring how people might be introduced to the service using prototypes"
+date: 2025-04-21
+tags:
+- alpha
+- prototyping
+---
+
+The personalised prevention platform (PPP) team has moved into an alpha phase.
+
+<figure>
+    <blockquote class="govuk-inset-text govuk-!-margin-left-0 govuk-!-padding-left-6" cite="https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works">
+      <p>Alpha is where you try out different solutions to the problems you learnt about during discovery.</p>
+      <p>Spend alpha building prototypes and testing different ideas. And do not be afraid to challenge the way things are done at the moment: alpha is a chance to explore new approaches.</p>
+      <p>You do not have to prototype the user’s entire wider journey.</p>
+      <p>You might not even want to prototype all of the transaction or element you’re working on: often it makes sense just to focus on the areas you think will be most challenging. This lets you do the minimum you need to test your riskiest assumptions.</p>
+    </blockquote>
+    <figcaption>From the <a href="https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works">GOV.UK Service Manual</a></figcaption>
+</figure>
+
+Amongst other things we’re building html prototypes to explore and test our hypotheses, and have just completed our first round, concerned with ‘onboarding’.
+
+## What we did
+
+Our main areas of interest in this round were:
+
+- do users understand what the service is and how it could help them?
+- how do users react to being offered the option of guidance versus going directly to services?
+- how willing are users to share their data to be matched to services, and what assumptions about data do they make?
+- how do users feel about the platform recommending areas to focus on versus their personal priorities?
+- do users retain focus on their health goal during an onboarding journey?
+
+To help us explore these topics, we built a mock journey that represented just the initial stages a potential journey:
+
+- entry points
+- starting to interact with a PPP
+- confirming and giving information
+
+We also made the following assumptions:
+
+- our scenario starts from an NHS.UK calculator
+- users are already somehow digitally registered ‘with the NHS’
+
+<figure class="govuk-!-margin-top-7">
+  <img src="prototype-flow.png" alt="Diagram depicting the prototype user journey using sequential screen grabs from left to right." style="width: 92vw; max-width: 960px;">
+  <figcaption>The prototype journey from left to right, from calculator result to a ‘finding services’ loading screen.</figcaption>
+</figure>
```

Comment:
Could use markdown here?

Reply:
I wanted the style adjustment of the full-width image and tweaked margin...?